<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="serialStartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>串口连接</string>
    </property>
   </widget>
   <widget class="QListWidget" name="antennaItemListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>611</width>
      <height>471</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>EPC内容</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>EPC长度</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>RSSI</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>天线ID</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="baudComboBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="inventoryButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始盘点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearInventoryButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>清空盘点</string>
    </property>
   </widget>
   <widget class="QComboBox" name="powerComboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>0.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7.0</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="setPowerButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>设置功率</string>
    </property>
   </widget>
   <widget class="QPushButton" name="networkStartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>网口连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setBaudButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>设置波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="inventoryCountLabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>510</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
